packages/zowe-explorer/src/utils/AuthUtils.ts

```diff
@@ -179,6 +179,125 @@ export class AuthUtils {
+                        const userIDIndex = toolTipList.findIndex((key) => key.startsWith("User: "));
+                        if (userIDIndex !== -1) {
+                            toolTipList[userIDIndex] = `User: ${profile.profile.user as string}`;
+                        } else {
+                            toolTipList.splice(authMethodIndex + 1, 0, `User: ${profile.profile.user as string}`);
+                        }
+                        break;
+                    }
+                    case Boolean(usingCertAuth): {
+                        toolTipList[authMethodIndex] = `Auth Method: Certificate Authentication`;
+                        break;
+                    }
+                    case !usingTokenAuth && !usingBasicAuth && !usingCertAuth: {
+                        toolTipList[authMethodIndex] = `Auth Method: Unknown`;
+                        const patternIndex = toolTipList.findIndex((key) => key.startsWith("Pattern: "));
+                        if (patternIndex !== -1) {
+                            toolTipList.splice(patternIndex, 1);
+                        }
+                        const pathIndex = toolTipList.findIndex((key) => key.startsWith("Path: "));
+                        if (pathIndex !== -1) {
+                            toolTipList.splice(pathIndex, 1);
+                        }
+                        const searchCriteriaIndex = toolTipList.findIndex((key) => key.startsWith("Owner: "));
+                        if (searchCriteriaIndex !== -1) {
+                            toolTipList.splice(searchCriteriaIndex, 1);
+                        }
+                        const jobIdIndex = toolTipList.findIndex((key) => key.startsWith("JobId: "));
+                        if (jobIdIndex !== -1) {
+                            toolTipList.splice(jobIdIndex, 1);
+                        }
+                    }
+                }
+                if (!usingBasicAuth) {
+                    const userIDIndex = toolTipList.findIndex((key) => key.startsWith("User: "));
+                    if (userIDIndex !== -1) {
+                        toolTipList.splice(userIDIndex, 1);
+                    }
+                }
+            }
+
+            if (usingTokenAuth || usingBasicAuth || usingCertAuth) {
+                switch (true) {
+                    case Boolean(sessionNode.fullPath): {
+                        const pathIndex = toolTipList.findIndex((key) => key.startsWith("Path: "));
+                        if (pathIndex === -1) {
+                            toolTipList.push(`Path: ${sessionNode.fullPath}`);
+                        } else {
+                            toolTipList[pathIndex] = `Path: ${sessionNode.fullPath}`;
+                        }
+                        break;
+                    }
+                    case sessionNode.description && (sessionNode.description as string).includes("Owner: "): {
```

Comment:
I see that we are using backticks (``` ` ```) on previous string definitions, which the prettier (or a linter rule) may advice against.

However, Since we are using it in other places, I do have a quick suggestion/trick that could save some type checking here.

```suggestion
                    case `${sessionNode.description}`.includes("Owner: "): {
```

The above will automatically apply/call the `toString()` method on the `description` property and it should always contain the `includes` since it's already a string 😋 

---

This is not a request for changes since the behavior should be the same as it is. 😋